README.md

```diff
@@ -58,4 +61,3 @@ However, Tox does use [TCP relays](/docs/TCP_Network.txt) as a fallback if it en
 ### Connecting & Communicating
 Every peer is represented as a [byte string](https://en.wikipedia.org/wiki/String_(computer_science)) (the public key [Tox ID] of the peer). By using torrent-style DHT, peers can find the IP of other peers by using their Tox ID. Once the IP is obtained, peers can initiate a [secure](/docs/updates/Crypto.md) connection with each other. Once the connection is made, peers can exchange messages, send files, start video chats, etc. using encrypted communications.
 
-
```

Comment:
One could add a statement about security vs privacy vs anonymity, because tox provides the first two by default but anonymity requires extra effort. This issue comes up pretty frequently (ie. can my friend see my IP address?).